dali/test/python/test_operator_crop.py

```diff
@@ -409,12 +434,12 @@ def __init__(self, device, batch_size, data_iterator, num_threads=1, device_id=0
         self.data_iterator = data_iterator
         self.layout = layout
 
-        self.crop = ops.Crop(device = self.device,
-                                crop = crop_shape,
-                                crop_pos_x = crop_x,
-                                crop_pos_y = crop_y,
-                                out_of_bounds_policy = out_of_bounds_policy,
-                                fill_values = fill_values)
+        self.crop = ops.Crop(device=self.device,
+                                crop=crop_shape,
```

Comment:
shouldn't those be aligned?

Reply:
This PR does not fix any alignment - these arguments keep the original alignment